test/UnitTest/Microsoft.AspNet.OData.Test.Shared/Batch/DefaultODataBatchHandlerTest.cs

```diff
@@ -612,4 +857,33 @@ public ITestActionResult Post([FromBody]BatchTestOrder order)
             return Created(order);
         }
     }
+#if NETCORE
```

Comment:
why is this only #if NETCORE?

Reply:
WebApi for AspNet and AspNetCore have different implementations of the batch handler that are not compatible. For example the batch handler in AspNet did not copy headers to individual requests in the first place, so it seems they worked differently. I added a test case for that. But I did not add the new header copying logic to AspNetCore.

Also, there's no test in that file that runs on both frameworks, they're either inside a `#if NETCORE` or `#if !NETCORE` block. So maybe it doesn't make that much sense to have these test cases in the shared project in the first place.